src/current/v23.1/restore.md

```diff
@@ -117,20 +117,21 @@ You can control `RESTORE` behavior using any of the following in the `restore_op
 
  Option                                                             | <div style="width:75px">Value</div>         | Description
  -------------------------------------------------------------------+---------------+-------------------------------------------------------
+`debug_pause_on`                                                    | `"error" `                                  |  Use to have a `RESTORE` [job]({% link {{ page.version.version }}/show-jobs.md %}) self pause when it encounters an error. The `RESTORE` job can then be [resumed]({% link {{ page.version.version }}/resume-job.md %}) after the error has been fixed or [canceled]({% link {{ page.version.version }}/cancel-job.md %}) to rollback the job. <br><br>Example: `WITH debug_pause_on='error'`
```

Comment:
Note: No new content! Alphabetized all of the options on the pages and just copied the the description of the `kms` option from the `BACKUP` page.